src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-row-view.ts

```diff
@@ -140,7 +141,7 @@ class GmailThreadRowView {
   _refresher: Kefir.Observable<any, any> | null | undefined;
   _subjectRefresher: Kefir.Observable<any, any> | null | undefined;
   _imageRefresher: Kefir.Observable<any, any> | null | undefined;
-  _counts: Counts | null | undefined;
```

Comment:
The ViewDriver(?) isn't part of the public API, and so it isn't subject to visibility concerns. `rg _counts` returns no matches after this change.